src/SprykerEco/Yves/Adyen/Theme/default/components/molecules/adyen-credit-card/adyen-credit-card.scss

```diff
@@ -8,6 +8,23 @@
             }
         }
 
+        .adyen-checkout__input {
```

Comment:
How these classes are related to this molecule?

Reply:
These classes are the inner stuff of the Adyen iframes. The goal of the molecule is rendering these iframes. This version of SDK have no opportunity to customize iframe styles from `ts`. Instead Adyen provides external css file and advise to override it to additional customization.

Reply:
Suggest to add a comment to describe it.